resources/crash.html

```diff
@@ -0,0 +1,8 @@
+<p>Servo crashed!</p>
+
+<!-- NOTE: unlike in Firefox and Chrome, this reloads POST as GET -->
+<!-- see whatwg/html#6600 + whatwg/html#3215 -->
+<button onclick="location.reload()">Reload</button>
+
+<pre><plaintext>
+${details}
```

Comment:
It looks like the [`<plaintext>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/plaintext) element is deprecated? Can this simply be:

```suggestion
<pre>${details}</pre>
```

Reply:
not quite, because if the crash details contain `<` or `&` then they will be interpreted as html. &lt;plaintext> solves that problem by permanently moving the parser to the PLAINTEXT state, and i figured that would be simpler than escaping the output.

Reply:
Hrm. Okay. I guess this is okay for now, but we should really consider adding some HTML escape functionality because it seems like `<plaintext>` is on the way out.